packages/lit-html/src/directives/async-append.ts

```diff
@@ -28,7 +28,7 @@ import {
 
 type Mapper<T> = (v: T, index?: number) => unknown;
 
-class AsyncAppendDirective extends AsyncDirective {
+class AsyncAppendDirectiveImpl extends AsyncDirective {
```

Comment:
I don't think these can be just `T` instead of `TImpl` since you're doing a type-only export.

Reply:
@e111077 did this not work?

```ts
export type {AsyncAppendDirective};
```

Reply:
Thanks for pointing this out in chat: s/I don't think/I think/

Reply:
@sorvell to collapse both convos into one thread

I implemented the following suggestion [here](https://github.com/Polymer/lit-html/pull/1627#discussion_r583988926) Would you like me to remove the renames?

Reply:
if `export type {AsyncAppendDirective}` works, do that.

Reply:
Yes, let's just do export type as @justinfagnani recommended.